Tasks/ContainerBuildV0/src/docker.ts

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+import path = require('path');
+import * as tl from "azure-pipelines-task-lib/task";
+import RegistryAuthenticationToken from "docker-common-v2/registryauthenticationprovider/registryauthenticationtoken";
+import ContainerConnection from "docker-common-v2/containerconnection";
+import { getDockerRegistryEndpointAuthenticationToken } from "docker-common-v2/registryauthenticationprovider/registryauthenticationtoken";
+
+export async function dockerBuildAndPush() {
+    let endpointId = tl.getInput("dockerRegistryServiceConnection");
+    let registryAuthenticationToken: RegistryAuthenticationToken = getDockerRegistryEndpointAuthenticationToken(endpointId);
+
+    // Connect to any specified container registry
+    let connection = new ContainerConnection();
+    connection.open(null, registryAuthenticationToken, true, false);
+
+    let resultPaths = "";
```

Comment:
I don't see output varaible declared in the task. This is not required. either add output variable or delete save output to a file.